<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <author>Generated by XSS. XKP 2011. E. Santos.</author>
    <class>Widget</class>

    	<widget class="QWidget" name="application">
    		<property name="geometry">
    			<rect>
    				<x>0</x>
    				<y>0</y>
    				<width>500</width>
    				<height>174</height>
    			</rect>
    		</property>
        <property name="windowTitle">
            <string>My first application</string>
        </property>
        	<widget class="QPushButton" name="pushButton_1">
        		<property name="geometry">
        			<rect>
        				<x>110</x>
        				<y>30</y>
        				<width>75</width>
        				<height>23</height>
        			</rect>
        		</property>
            <property name="text">
                <string>Click me</string>
            </property>
            <property name="enabled">
                <bool>1</bool>
            </property>
        	</widget>
        	<widget class="QPushButton" name="pushButton_2">
        		<property name="geometry">
        			<rect>
        				<x>210</x>
        				<y>30</y>
        				<width>91</width>
        				<height>23</height>
        			</rect>
        		</property>
            <property name="text">
                <string>Change text</string>
            </property>
            <property name="enabled">
                <bool>false</bool>
            </property>
        	</widget>
        	<widget class="QLabel" name="label_1">
        		<property name="geometry">
        			<rect>
        				<x>170</x>
        				<y>60</y>
        				<width>120</width>
        				<height>16</height>
        			</rect>
        		</property>
            <property name="text">
                <string>Here is the text</string>
            </property>
            <property name="alignment">
                <enum>Qt::AlignLeft</enum>
            </property>
            <property name="enabled">
                <bool>1</bool>
            </property>
        	</widget>
        	<widget class="QWidget" name="holder">
        		<property name="geometry">
        			<rect>
        				<x>130</x>
        				<y>80</y>
        				<width>300</width>
        				<height>60</height>
        			</rect>
        		</property>
            <property name="enabled">
                <bool>1</bool>
            </property>
            	<widget class="QLabel" name="div_label_1">
            		<property name="geometry">
            			<rect>
            				<x>110</x>
            				<y>10</y>
            				<width>80</width>
            				<height>16</height>
            			</rect>
            		</property>
                <property name="text">
                    <string>Inside div holder</string>
                </property>
                <property name="alignment">
                    <enum>Qt::AlignLeft</enum>
                </property>
                <property name="enabled">
                    <bool>1</bool>
                </property>
            	</widget>
            	<widget class="QPushButton" name="pushButton_3">
            		<property name="geometry">
            			<rect>
            				<x>null</x>
            				<y>null</y>
            				<width>60</width>
            				<height>40</height>
            			</rect>
            		</property>
                <property name="text">
                    <string>--)</string>
                </property>
                <property name="enabled">
                    <bool>1</bool>
                </property>
            	</widget>
            	<widget class="QPushButton" name="pushButton_4">
            		<property name="geometry">
            			<rect>
            				<x>null</x>
            				<y>null</y>
            				<width>60</width>
            				<height>40</height>
            			</rect>
            		</property>
                <property name="text">
                    <string>(--</string>
                </property>
                <property name="enabled">
                    <bool>1</bool>
                </property>
            	</widget>
            	<widget class="QLineEdit" name="edtText">
            		<property name="geometry">
            			<rect>
            				<x>90</x>
            				<y>30</y>
            				<width>130</width>
            				<height>23</height>
            			</rect>
            		</property>
                <property name="text">
                    <string>write me!</string>
                </property>
                <property name="alignment">
                    <enum>Qt::AlignRight</enum>
                </property>
                <property name="enabled">
                    <bool>1</bool>
                </property>
            	</widget>
        	</widget>
        	<widget class="QPushButton" name="pushButton_5">
        		<property name="geometry">
        			<rect>
        				<x>null</x>
        				<y>null</y>
        				<width>60</width>
        				<height>30</height>
        			</rect>
        		</property>
            <property name="text">
                <string>Button at Bottom of window</string>
            </property>
            <property name="enabled">
                <bool>1</bool>
            </property>
        	</widget>
    	</widget>

    <layoutdefault spacing="3" margin="5"/>
    <resources/>
    <connections/>
</ui>

